### PR TITLE
T10313: Add custom values to wgCargoAllowedSQLFunctions for sagan4wiki and sagan4alphawiki

### PR DESCRIPTION
**Pull request by redbluegreenhat**
Requested on Phabricator: https://phabricator.miraheze.org/T10313.

### CODE REVIEW

**Comment by Universal-Omega** [comment]
This can be in LocalWiki.php due to how large it is so that the whole thing doesn't need defined again.

File: LocalWiki.php [hunk 1]
```diff
@@ -292,6 +292,12 @@ function onBeforeInitialize( Title &$title, $unused, OutputPage $output, User $u
 			}
 		}
 
+		break;
+	case 'sagan4wiki':
```
Comment (Universal-Omega) [inline LocalWiki.php:296]: 2 case statements here will combine them since it is the same thing.

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: LocalWiki.php (hunk 1 above)
Comment (redbluegreenhat) [reply LocalWiki.php:296]: ?

Comment (Universal-Omega) [reply LocalWiki.php:296]: Like https://github.com/miraheze/mw-config/blob/1713daaa1315f4a79cb8cdd7a2b93f77234deebc/LocalWiki.php#L310-L311

Comment (redbluegreenhat) [reply LocalWiki.php:296]: I didn't even know you could do that in PHP.

**Review by Universal-Omega — APPROVED** [review]

**Pull request merged**